src/test/resources/reference.conf

```diff
@@ -39,8 +39,8 @@ chicago {
           bindPort = 12001
         }
       }
-      rpc = ${xio.serverTemplate} {
-        name = "Chicago RPC Server"
+      paxos = ${xio.serverTemplate} {
```

Comment:
Did we forget to deprecate this file? We should be using `src/main/resources/reference.conf`